File: Java/Baju.java
public class Baju extends Aksesoris {
  private String untuk;
  private int size;
  private String merk;

  public Baju() {
  }

  public Baju(int id, String name, String category, int price, int size, String jenis, String warna, String bahan, String untuk, String merk) {
    super(id, name, category, price, jenis, warna, bahan);
    this.size = size;
    this.untuk = untuk;
    this.merk = merk;
  }

  void setUntuk(String untuk) {
    this.untuk = untuk;
  }

  void setSize(int size) {
    this.size = size;
  }

  void setMerk(String merk) {
    this.merk = merk;
  }

  String getUntuk() {
    return untuk;
  }

  int getSize() {
    return size;
  }

  String getMerk() {
    return merk;
  }

  void showBaju(){
    System.out.printf("%-5d %-15s %-10s %-10d %-10d %-10s %-10s %-10s %-10s %-5s\n",
    getId(), getName(), getCategory(), getPrice(), getSize(), getJenis(), getWarna(), getBahan(), getUntuk(), getMerk());
  }
}

File: CPP/Baju.cpp
#include <bits/stdc++.h>
#include "Aksesoris.cpp"
using namespace std;

class Baju : public Aksesoris
{
protected:
    string untuk;
    int size;
    string merk;

public:
    Baju()
    {
    }

    Baju(int id,
        string name,
        string category,
        int price, 
        int sz, 
        string jenis, 
        string warna, 
        string bahan, 
        string utk, 
        string mrk) : Aksesoris(id, name, category, price, jenis, warna, bahan)
    {
        this->untuk = utk;
        this->size = sz;
        this->merk = mrk;
    }

    void setUntuk(string utk)
    {
        this->untuk = utk;
    }

    string getUntuk()
    {
        return this->untuk;
    }

    void setSize(int sz)
    {
        this->size = sz;
    }

    int getSize()
    {
        return this->size;
    }

    void setMerk(string mrk)
    {
        this->merk = mrk;
    }

    string getMerk()
    {
        return this->merk;
    }

    void showBaju()
    {
        cout << left
             << setw(5) << getId()
             << setw(15) << getName()
             << setw(10) << getCategory()
             << "Rp " << fixed << getPrice() << "  "
             << setw(10) << getJenis()
             << setw(10) << getWarna()
             << setw(10) << getBahan()
             << setw(10) << getUntuk()
             << setw(10) << getSize()
             << setw(5) << getMerk()
             << endl;
    }

    ~Baju()
    {

    }
};
